src/main/java/org/fundacionjala/coding/daniel/CamelCaseMethod.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.coding.daniel;
+
+import java.util.StringJoiner;
+
+/**
+ * All words must have their first letter capitalized without spaces.
+ */
+public class CamelCaseMethod {
+
+    /**
+     * Methos that puts the words together and capitalizes the first letter of each word.
+     *
+     * @param cadena string of words.
+     * @return cadena unida con la primera palabra en mayuscula.
+     */
+    public String camelCase(String cadena) {
+        StringJoiner joiner = new StringJoiner("");
```

Comment:
For this scenario is not necessary to use a StringJoiner, it is enough with a StringBuilder